registration/templates/registration/password_reset_complete.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% trans "You may now" %} <a href="{{ login_url }}">{% trans "log in" %}</a>.
```

Comment:
I'm glad to hear that, you should put both of your names on the README
So ppl knows who is maintaining